modules/javafx.controls/src/test/java/test/javafx/scene/control/AlertTest.java

```diff
@@ -92,7 +92,8 @@ private void showAndHideDialog(Dialog<?> dialog, boolean normalClose) {
                         Platform.runLater(() -> getStage(dialog).close());
                     }
                 } catch (InterruptedException e) {
-                    e.printStackTrace();
+                    // fail(e);
+                    throw new AssertionError(e);
```

Comment:
I left the commented out "fail(e)" as a possibly better alternative once these tests are updated to JUnit 5, which is under review at PR #1569. Depending on the timing of integrating that PR and this one, I will either merge in master (I've already done a test merge and there are no merge conflicts) and update these calls to use "fail" or file a follow-on bug.